app/src/main/java/org/mozilla/fenix/tabstray/AccessibleNewTabButtonBinding.kt

```diff
@@ -26,13 +27,14 @@ import org.mozilla.fenix.utils.Settings
 class AccessibleNewTabButtonBinding(
     private val store: TabsTrayStore,
     private val settings: Settings,
-    private val newTabButton: ImageButton,
+    private val actionButton: ImageButton,
     private val browserTrayInteractor: BrowserTrayInteractor
 ) : AbstractBinding<TabsTrayState>(store) {
 
+    @SuppressLint("MissingSuperCall") // early return is intentional.
```

Comment:
Perhaps the comment should go before the early.

Reply:
Fixed the comment instead to be more accurate. 👌